# Análise - Airbnb Latam

Você trabalha na divisão América Latina do Airbnb e está comparando a operação do Airbnb nos 4 principais países da empresa na região: Brasil, México, Argentina e Colômbia. Um dos objetivos da divisão é aumentar a quantidade de diárias reservadas por turistas estrangeiros nas hospedagens disponíveis na América Latina.

In [1]:
# Importando as libs
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot

In [2]:
# Caminho do csv
path = 'C:/Users/João Henrique/Documents/Projetos/airbnb-analysis/data/Airbnb - Imoveis.csv'

In [3]:
# Ler csv
df = pd.read_csv(path)

In [4]:
# Mostrando o df
df.head()

Nome do local   Localização       País Tipo de acomodação Classificação  \
0  Hostel Montanha    San Andrés   Colômbia             Hostel    5 estrelas   
1         Mirliban  Buenos Aires  Argentina        Apartamento    5 estrelas   
2  Estrela Dourada   Guadalajara     México              Hotel    5 estrelas   
3   Mar Azul Beach     Fortaleza     Brasil              Hotel    5 estrelas   
4  Montanha Mágica      Curitiba     Brasil             Quarto    4 estrelas   

              Comodidades Políticas de cancelamento   Preço  \
0    Wi-Fi, café da manhã              7 dias antes  $30,00   
1  Wi-Fi, ar condicionado                  72 horas  $10,00   
2       Piscina, recepção              reembolsável  $14,00   
3         Wi-Fi, armários            48 horas antes  $35,00   
4  Piscina, churrasqueira              7 dias antes  $48,00   

   Número de reservas no ano  Número de Fotos  
0                         11                5  
1                         25                7  
2                         73               12  
3                        162                2  
4                        200                2

In [5]:
# Tamanho do df (linhas x colunas)
df.shape

(49, 10)

In [6]:
# Informações do df
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 49 entries, 0 to 48
Data columns (total 10 columns):
 #   Column                     Non-Null Count  Dtype 
---  ------                     --------------  ----- 
 0   Nome do local              49 non-null     object
 1   Localização                49 non-null     object
 2   País                       49 non-null     object
 3   Tipo de acomodação         49 non-null     object
 4   Classificação              49 non-null     object
 5   Comodidades                49 non-null     object
 6   Políticas de cancelamento  49 non-null     object
 7   Preço                      49 non-null     object
 8   Número de reservas no ano  49 non-null     int64 
 9   Número de Fotos            49 non-null     int64 
dtypes: int64(2), object(8)
memory usage: 4.0+ KB


## 1. Analise as principais métricas aos preços das diárias nos 4 países e compartilhe os 3 principais insights que você teve ao compará-las entre os países.

In [7]:
# Convertendo o tipo da coluna preço para float
df['Preço'] = df['Preço'].str.replace('$', '', regex=False).str.replace(',', '.', regex=False).astype(float)

In [ ]:
# Verificando as métricas de preço dos 4 países
df.groupby('País')['Preço'].describe()

count        mean        std   min     25%    50%     75%    max
País                                                                       
Argentina   10.0  168.100000  78.297793  10.0  137.00  183.5  223.75  272.0
Brasil      15.0  163.200000  81.600245  35.0  101.50  170.0  218.50  300.0
Colômbia    14.0  145.928571  61.537311  30.0  114.75  160.0  193.50  249.0
México      10.0  152.700000  72.562693  14.0  139.00  156.5  183.25  289.0

Apesar de ser um dos países com menor quantidade de reservas (10) a Argentina se mostra o país com a maior média e mediana e um dos menores desvio padrão, isso indica que os valores não estão tão dispersos da média.  
O Brasil é o país com mais reservas (15) e uma média de $163.20, vemos que o desvio padrão é exatamente metade da média, isso nos mostra que temos uma grande dispersão em relação a média, também podemos ver que é o país
com a maior reserva $300.00.  
Apesar da Colômbia ser o 2 país com mais reservas (14), é o país com menor média e com menor desvio padrão, ou seja, os valores das reservas estão próximos a média, sua maior reserva $249.00 ainda é a menor se comparado aos outros
países, como a quantidade de reservas é alta, podemos tirar que é um dos países custo-benefício de se visitar.

In [25]:
std = df.groupby('País')['Preço'].std()
mean = df.groupby('País')['Preço'].mean()
cv = std/mean * 100
cv

País
Argentina    46.578104
Brasil       50.000150
Colômbia     42.169474
México       47.519773
Name: Preço, dtype: float64